### ISSUE TITLE
List in /tests page not scrollable

### ISSUE THREAD

**Issue by flyswatter** [sglanzer/ember-cli-blanket#49]
If I call `ember serve` and navigate to my `/tests` route and click `show coverage`, the coverage tests are shown in an element, but if the list is long, it will spill out of view, and is not scrollable.

**Comment by flyswatter** [comment]
I've searched the entire project for `.css` files, to see where this style would be defined, but only found the `tests/dummy/app/styles/app.css` file.  Any tips where the `#blanket-main` styles are declared?

**Comment by chrisvdp** [comment]
sounds like it didn't install properly. They should be added inline to tests/index.html

**Comment by chrisvdp** [comment]
<style>#blanket-main { position: relative; z-index: 99999; }</style>

**Comment by jschilli** [comment]
Dan,  it's in your tests.html. Or should be. Inserted by index.js

Sounds like you did an npm install whereas the blanket file install and css touchup happen in after install

Not at the computer right now. But I can take a look at this and see what alternatives we have

-jeff

**Comment by flyswatter** [comment]
> Sounds like you did an npm install

I actually did the `ember install:addon ember-cli-blanket` right off the README.

> sounds like it didn't install properly. They should be added inline to tests/index.html

They *are* added, their div element just isn't scrollable.

I got it doing what I wanted by adding this CSS to my project:

```
#blanket-main{ overflow-y: scroll; }
```

But sounds like this was an install error?  Leaving this open so we can explore what went wrong.

**Comment by flyswatter** [comment]
Oh yes, `#blanket-main { position: relative; z-index: 99999; }` was inserted into my `tests/index.html` file, but it still wasn't scrollable until I added `overflow-y:scroll`.

**Comment by chrisvdp** [comment]
what browser?

**Comment by flyswatter** [comment]
Chrome Version 42.0.2311.90 (64-bit)

**Comment by chrisvdp** [comment]
Running the same and don't have any issues. I am using mocha though.

**Comment by flyswatter** [comment]
This may be because I'm running a css-reset, and `ember-cli-blanket` is assuming some default style behavior.

This would present a decent question to consider:  Should this addon expect a certain style context, or should it provide the styles that it relies on?

**Comment by edmundito** [comment]
@flyswatter Right, it seems that the CSS in your app may be interfering with the css for the test runner. I had to add this to `tests/index.html`

```css
body { overflow-y: scroll; }
```

...because we were doing a similar reset in our app. Your suggestion helped us fix it.

**Comment by flyswatter** [comment]
It's interesting that the app's reset can extend outside itself, but I guess a reset indiscriminately applies to `html, body`, so why shouldn't it?

**Comment by jschilli** [comment]
@flyswatter recommendation here? or close?

**Comment by flyswatter** [comment]
If it doesn't break others' formats if say add the Css that we did, but you would want to test it on some other projects first.

Otherwise a note in the readme I guess would do.
